packages/example-log-extension/test/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--recursive
```

Comment:
This can be dropped since `package.json` uses `mocha.opts` from `@loopback/build`.